src/probability_theory/integration.lean

```diff
@@ -0,0 +1,118 @@
+/-
+Copyright (c) 2021 Martin Zinkevich. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Martin Zinkevich
+-/
+import measure_theory.integration
+import probability_theory.independence
+
+/-!
+# Integration in Probability Theory
+
+Integration results for independent random variables. Specifically, for two
+independent random variables X and Y over the extended non-negative
+reals, `E[X * Y] = E[X] * E[Y]`, and similar results.
+-/
+
+noncomputable theory
+open set probability_theory
+open_locale ennreal
+
+variables {α : Type*}
+
+namespace measure_theory
```

Comment:
Maybe use the probability_theory namespace instead of measure_theory, since the lemmas are about independence?

Reply:
I am indifferent. It ironically uses more theorems from measure_theory, but it is easy enough to just open up that namespace. Floris?

Reply:
Honestly, I would vote against the `probability_theory` namespace even existing. It just means that we have to `open measure_theory probability_theory` above every file about probabilities, instead of just `open measure_theory` if we keep everything in the namespace `measure_theory`. 

I'm also fine with keeping `probability_theory`, and then I think we should indeed use the `probability_theory` here.

Reply:
Okay, I added this to probability_theory for now. We can refactor later.